exampleSite/content/about.es.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Hugo tip: you can now put each language in its own content folder and drop the ".es" naming. Might not be a good idea for smaller sites.

Reply:
Hi @bep , is this anywere documented? I haven't found any of this in the multilingual mode documentation of Hugo: https://gohugo.io/content-management/multilingual/

Reply:
Hmm... Seems I have a back log of stuff to document. http://gohugo.io/news/0.38-relnotes/ Also see the Gohugoio source for an example.

Reply:
Thank you @bep!!

And thank you @pmviva for your extensive work and patience :bowing_woman: